ci/scripts/shipit

```diff
@@ -47,15 +47,15 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ###############################################################
 
 cd ${REPO_ROOT}
-RELEASE_NAME=$(bosh int config/final.yml --path /final_name)
+RELEASE_NAME=$(bosh int config/final.yml --path /name)
 
 cat > config/private.yml <<YAML
 ---
 blobstore:
-  provider: s3
   options:
-    access_key_id: ${AWS_ACCESS_KEY}
-    secret_access_key: ${AWS_SECRET_KEY}
+    credentials_source: static
+    json_key: |
+$(PAD=$(printf ' %.0s' {1..6}); sed -E 's/^(.*)$/'"$PAD"'\1/g' <<<"$GCP_SERVICE_KEY")
```

Comment:
That's a bit unreadable, some documentation might be nice?
Maybe extract it to a variable outside of YAML first?